styles/abstracts/_theme.scss

```diff
@@ -30,6 +30,8 @@ $themes: (
     'primary-shadow': 4px 4px black,
     'disabled': #f3f3f3,
     'card-border-color': #999999,
+    'collection-search-tag-bg': #B6CBFF,
+    'blue-tag-hover-bg': #E8EDFB,
```

Comment:
we have these color already
k-blueaccent
k-blueLight

possible to use them instead of defining new ones?

Reply:
We could not use k-bluexx because it did not fit the color on both light and dark mode as figma shown. That means I have to create a new theme name.

Reply:
how about giving it some general name? in case it will be reused in the future?

Reply:
Yes, I have renamed them.

Reply:
thank you :)